content/en/docs/reference/kubectl/overview.md

```diff
@@ -545,4 +546,4 @@ Current user: plugins-user
 
 * To find out more about plugins, take a look at the [example cli plugin](https://github.com/kubernetes/sample-cli-plugin).
 
-
+ 
```

Comment:
Deleting L549, making only one blank line (L548) exist at the end of file, may be a good change. 😊

```suggestion
```

Reply:
Yeah sure and thanks for your review :)